runtime/gc_base/GCExtensions.cpp

```diff
@@ -297,11 +297,12 @@ MM_GCExtensions::releaseNativesForContinuationObject(MM_EnvironmentBase* env, j9
 
 	if (verify_continuation_list == continuationListOption) {
 		J9VMContinuation *continuation = J9VMJDKINTERNALVMCONTINUATION_VMREF(vmThread, objectPtr);
-		jboolean finished = J9VMJDKINTERNALVMCONTINUATION_FINISHED(vmThread, objectPtr);
+
+		ContinuationState *continuationStatePtr = VM_VMHelpers::getContinuationStateAddress(vmThread, objectPtr);
```

Comment:
would compiler force you to have:

ContinuationState volatile *continuationStatePtr =...

Reply:
yes